VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitVideoFragment.java

```diff
@@ -833,6 +832,9 @@ public void onConfigurationChanged(Configuration newConfig) {
                 playerContainer.requestLayout();
             }
         }
+        if (playerFragment != null) {
+            playerFragment.handleOnResume();
```

Comment:
Instead of hacking the effect by invoking the resume callback, you could just implement the `onConfigurationChanged()` callback in `PlayerFragment` itself instead, and call `player.setFullScreen(isScreenLandscape());` directly.